docker/examples/sensorthings/README.md

```diff
@@ -5,27 +5,48 @@ This folder contains a Docker Compose configuration necessary to setup an exampl
+
 ## SensorThings Build options
 
-There are three example STA endpoints. To switch between examples, the `pygeoapi.config.yml` file used in the docker
-compose needs to be changed.
+There are three example SensorThings API (STA) endpoints available. To switch between examples, you need to change the `pygeoapi.config.yml` file used in the Docker Compose file.
+
+1. [**brgm.config.yml**](brgm.config.yml): Configures a `pygeoapi` server to serve water quality data from BRGM (Bureau de Recherches Géologiques et Minières), the French Geological Survey.
+
+2. [**usgs.config.yml**](usgs.config.yml): Configures a `pygeoapi` server to serve data from the United States Geological Survey (USGS).
+
+3. [**docker-compose.yml**](docker-compose.yml): Defines the Docker Compose configuration for the `pygeoapi` server. It specifies the Docker image to use, the ports to expose, and the volumes to mount.
+
+### Additional details
+
+- The BRGM water quality endpoint provides access to water quality data from BRGM, the French Geological Survey.
 
-- The first, [brgm.sta.pygeoapi.config.yml](brgm.sta.pygeoapi.config.yml) creates a `pygeoapi` server serving the BRGM water quality endpoint. 
+- The USGS data endpoint provides access to data from the United States Geological Survey (USGS).
 
-- The second, [iow.sta.pygeoapi.config.yml](iow.sta.pygeoapi.config.yml) creates a `pygeoapi` server hosting example IoW endpoint with URIs. 
+### Which build option should I choose?
 
-- The final config, [sta.pygeoapi.config.yml](sta.pygeoapi.config.yml) creates a `pygeoapi` server inside of a muti-container Docker app,
-and linked to an empty [FROST server](https://fraunhoferiosb.github.io/FROST-Server/). To use this configuration, 
-uncomment Lines 40 - 70 of the docker-compose.yml file in addition to changing the pygeoapi config. The 
-database can be populated following a workflow similar to that of the populator script for build testing, `load_sta_data.py`.
-`Note: The pygeoapi server will fail to build until the STA server has been populated, thus the addition of always restart`
+The best build option for you will depend on your specific needs. If you are interested in accessing water quality data from the BRGM, then you should choose the `brgm.config.yml` file. If you are interested in accessing data from the USGS, then you should choose the `usgs.config.yml` file.
+
+If you are not sure which build option to choose, you can start with the `brgm.config.yml` file. This is the simplest build option and it provides access to a _real-world dataset_.
 
 ## SensorThings Usage
 
 After editing the [docker-compose.yml](docker-compose.yml) file appropriately:
 
-```
+```bash
 docker compose up [-d]
 ```
 
-Navigate to `localhost:5000`.
\ No newline at end of file
+This will create and start the pygeoapi container. You can then access the SensorThings API server at http://localhost:5000.
```

Comment:
Actually, this is not the STA server, but the pygeoapi server, republishing SensorThings data,